### ISSUE TITLE
Add chocolatey package 

### ISSUE THREAD

**Issue by NyCodeGHG** [Chatterino/chatterino2#1981]
**What should be added?**
I would like to use [Chocolatey](https://chocolatey.org/) to install and update Chatterino. It's a package manager for windows like apt or pacman on Linux.

**Why should it be added?**
For more advanced users it's faster to install when they simply can open up a powershell and type
```bash
choco install chatterino
```
than open up the website, download the installer and so.

Let me know what you think! 😄

Greetings, Nico

**Comment by fourtf** [comment]
You have the job!

**Comment by alazymeme** [comment]
I have finished this, just awaiting review. @fourtf if you create a chocolatey account and add the API key to the secrets in github, I should be able to setup a github action that automatically pushes a build when you update the stable branch.

**Comment by fourtf** [comment]
@alazymeme I'll update it myself, ty for creating the package

**Comment by alazymeme** [comment]
@fourtf I got a response from the reviewers. They basically said: use install arguments to make the installation silent or use autohotkey as a dependency and use it to click the options as necessary. Are there any command arguments that would allow a check for the vcredists and leave the other two options unchecked? Or are you perhaps able to compile a custom installer for chocolatey that checks for the vcredists and installs them automatically? 🤔

**Comment by alazymeme** [comment]
After quite a few changes, I've got a package that has passed most of the checks. Just needs to be accepted by the reviewer, then people will be able to download it: https://chocolatey.org/packages/chatterino

In the end we had to deselect the vcredist option in the installer and run a verysilent install in order for the checks to pass. I've added vcredist chocolatey package as a dependency (which does silently install) so there should be no problems.

Once this package has been accepted, I might push all the older versions to the page for version tracking (since the release notes have all the versions from the changelog page).
